terraform/ec2-test.tf

```diff
@@ -0,0 +1,101 @@
+module "test_vpc" {
+  source = "github.com/terraform-aws-modules/terraform-aws-vpc"
+
+#   name = "${var.vpc_name}"
+
+#   cidr = "${var.vpc_cidr}"
+#   public_subnets  = ["${var.app_public_subnet_cidrs}"]
+#   private_subnets = ["${var.app_private_subnet_cidrs}"]
+#   database_subnets = ["${var.database_subnet_cidrs}"]
+
+#   enable_nat_gateway = "${var.enable_nat_gateway}"
+#   enable_dns_hostnames = "${var.enable_dns_hostnames}"
+#   enable_dns_support = "${var.enable_dns_support}"
+
+#   azs = ["${var.aws_az1}", "${var.aws_az2}"]
+
+#   create_database_subnet_group = "${var.create_database_subnet_group}"
```

Comment:
Did you mean to leave all these commented out?

Reply:
Nope, meant to remove them. Fixed ;)